test/throws.js

```diff
@@ -0,0 +1,13 @@
+'use strict';
+
+var expect  = require('chai').expect;
+var lost    = require('../lost');
+var postcss = require('postcss');
+var CssSyntaxError = require('postcss').CssSyntaxError;
+
+module.exports = function throws( input, message, opts ) {
+  var processor = postcss([lost(opts)]);
+  expect(function(){
+    return processor.process(input).css;
+  }).to.throw(CssSyntaxError, message);
+};
```

Comment:
Nitpicking absolutely fine 👍  I generally use PHPStorm as my IDE which does put in the newline at the end of files, but I'm just trying out VSCode for javascript work and unfortunately doesn't use `.editorconfig` out of the box (as I've just found out), so completely missed this!

I've updated the file in question now.